test/constrained-generics/ucol/neelam/min_concept_no_impl_stmt.good

```diff
@@ -0,0 +1,2 @@
+min_concept_no_impl_stmt.chpl:18: error: unresolved call 'minFn(3, 4)'
+min_concept_no_impl_stmt.chpl:9: note: candidates are: minFn(x: ?T, y: T)
```

Comment:
Hi Vass, it looks like the tests that go with the two _no_impl_ .good files were never committed.  Do you still have them?